src/nxdoc/web-ui/web-ui-release-notes/web-ui-release-notes-3-0-6.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Nuxeo Web UI for LTS 2021 Release Notes
+description: Discover changes brought in our recent Nuxeo Web UI updates.
+review:
+    comment: ''
+    date: '2021-09-02'
+    status: ok
+toc: true
+labels:
+tree_item_index: 998
+hidden: true
+---
+
+{{{multiexcerpt 'matching-notes' page='web-ui-release-notes'}}}
+
+## What's New in Web UI for LTS 2021 (version 3.0.6)
+
+{{! multiexcerpt name='web-ui-updates'}}
+
+### Execute Actions in Bulk
+
+This release introduces a first iteration for bulk actions. Users can now select all documents from a results listing both in a search or browsing context, then execute an action on them.
+
+Bulk actions can be executed on most default actions from Web UI like adding documents to a collection or publishing them. It is also possible to [create your custom bulk actions using Nuxeo Studio]({{page page='howto-create-bulk-actions-studio' space='studio'}}).
+
+This feature is currently disabled by default; you can activate it by adding the property `nuxeo.selection.selectAllEnabled=true` in your [nuxeo.conf](configuration-parameters-index-nuxeoconf) file. Learn more with the [bulk actions documentation]({{page page='web-ui-bulk-actions'}}).
```

Comment:
Do we want to make this public already?

Reply:
Critical issues we had have been fixed in 3.0.6 and 2021.8 so why not?